lib/lookup.json

```
@@ -1,4 +1,11 @@
 {
+  "@babel/core": {
+    "maintainers": "nicolo-ribaudo",
```

```suggestion
    "maintainers": ["nicolo-ribaudo", "JLHwung", "liuxingbaoyu"],
```

@JLHwung @liuxingbaoyu Is it ok to add you two here, in case I'm not available when there is a problem?

---

Sure! Happy to do so.